src/dodal/beamlines/training_rig.py

```diff
@@ -24,6 +28,14 @@
 set_log_beamline(BL)
 set_utils_beamline(BL)
 
+set_path_provider(
+    StaticVisitPathProvider(
+        BL,
+        Path("/exports/mybeamline/data"),
+        client=LocalDirectoryServiceClient(),
```

Comment:
n.b. this means the file number will reset each time blueapi is restarted @callumforrester